dietpi/func/dietpi-set_hardware

```diff
@@ -1896,8 +1902,14 @@ _EOF_
 				fi
 			;;
 
-			# Intel SST DSP
-			'intel-sst-dsp') G_AG_CHECK_INSTALL_PREREQ 'firmware-intel-sound';;
+                        # Intel SST DSP
+                        'intel-sst-dsp')
+
+                                local aPackages=('firmware-intel-sound')
+                                (( $G_DISTRO >= 6 )) && aPackages+=('firmware-sof-signed') #Buster additions
```

Comment:
Available since Bullseye, not Buster 😉:
```suggestion
                                (( $G_DISTRO > 5 )) && aPackages+=('firmware-sof-signed') # Available since Bullseye
```